spring-cloud-generator/java_gapic_spring.bzl

```diff
@@ -0,0 +1,146 @@
+# Copyright 2022 Google LLC
```

Comment:
New files in this repo need to be have `2023` in the copyright header

Reply:
Also the copyright format is different from [other files](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/main/spring-cloud-gcp-storage/src/main/java/com/google/cloud/spring/storage/GoogleStorageProtocolResolver.java#L1C1-L15) in this repo

Reply:
Per discussion - updated license header year to 2023 ([eefaeb6](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/1686/commits/eefaeb6aa3b2c0b5c84d420e82bce1c8df204195)) for migrated files that previously lived on branch, but not published to gapic-generator-java main.